unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -955,23 +960,34 @@ mod tests {
 
     #[test]
     fn evaluates_host_name_constraint_correctly() {
-        std::env::set_var("HOSTNAME", "DOS");
+        std::env::set_var("hostname", "DOS");
 
         let rule = compile_rule("hostname in [\"DOS\"]").unwrap();
         let context = Context::default();
         assert!(rule(&context));
 
-        std::env::remove_var("HOSTNAME");
+        std::env::remove_var("hostname");
     }
 
     #[test]
     fn evaluates_host_name_to_false_when_missing_hostname_values() {
-        std::env::set_var("HOSTNAME", "DOS");
+        std::env::set_var("hostname", "DOS");
 
         let rule = compile_rule("hostname in [\"\"]").unwrap();
         let context = Context::default();
         assert!(!rule(&context));
 
-        std::env::remove_var("HOSTNAME");
+        std::env::remove_var("hostname");
+    }
+
+    #[test]
+    fn hostname_constraint_ignores_casing() {
+        std::env::set_var("hostname", "DoS");
+
+        let rule = compile_rule("hostname in [\"dOS\", \"pop-os\"]").unwrap();
```

Comment:
is this dos with O as a number 0? i think that is confusing

Reply:
No? That's the letter "O" not the number "0".

I can change it if you think the font is misleading?

Reply:
Swapped out for a weirdly cased `darwin`. Thanks I can never unsee this now. I'm always going to question PRs on GH that include either O or 0